boards/arm/bl654_dvk/Kconfig.defconfig

```diff
@@ -42,7 +42,7 @@ endif # SPI
 
 if USB
 
-config USB_NRF52840
+config USB_NRF52
```

Comment:
This  should be `USB_NRF` or `USB_NRFX`, that's what we do for the rest of the hardware. CC @anangl

Reply:
yes, `USB_NRFX` will be consistent with what we do for other drivers, plus the options that follow in this file start with `USB_NRFX_`